docs/contributing/contributing-checklist.md

```diff
@@ -4,118 +4,117 @@ title: Contribution Checklist
 
 Following these instructions helps us make sure the code review and merge process go smoothly.
 
-Before submitting a pull request
-Once your code is ready, please go through the following checklist before submitting a pull request.
-
-## 1. Have you signed the CLA?
+###Before submitting a pull request
 
-A Contributor License Agreement helps guarantee that contributions to Great Expectations will always remain free and open.
+Once your code is ready, please go through the following checklist before submitting a pull request.
 
-Please see Contributor license agreement (CLA) for more information and instructions for how to sign the CLA the first time you contribute to Great Expectations.
+#### 1. Have you signed the CLA?
 
-If you’ve included your (physical) mailing address in the CLA, we’ll send you a personalized Great Expectations mug once your first PR is merged!
+* A Contributor License Agreement helps guarantee that contributions to Great Expectations will always remain free and open.
 
-## 2. Have you followed the Style Guide for code and comments?
+* Please see [Contributor license agreement](https://docs.greatexpectations.io/en/0.13.8/contributing/miscellaneous.html#contributing-cla) (CLA) for more information and instructions for how to sign the CLA the first time you contribute to Great Expectations.
 
-The Style Guide is here.
+* If you’ve included your (physical) mailing address in the CLA, we’ll send you a personalized Great Expectations mug once your first PR is merged!
 
-Thanks for helping us keep the codebase and documentation clean and consistent, so that it’s easier to maintain it as a community!
+#### 2. Have you followed the Style Guide for code and comments?
 
-If your PR contains only changes to contrib directory (community contributed Expectations), review your PR against this Acceptance Checklist
+* The [Style Guide](/docs/contributing/contributing-style) is here.
 
-## 3. Is your branch up to date with upstream/develop?
+* Thanks for helping us keep the codebase and documentation clean and consistent, so that it’s easier to maintain it as a community!
 
-Update your local repository with the most recent code from the main Great Expectations repository.
+* If your PR contains only changes to `contrib` directory (community contributed Expectations), review your PR against this Acceptance Checklist
```

Comment:
Looking into this further [Acceptance Checklist](https://docs.greatexpectations.io/en/latest/contributing/contrib_pull_request_acceptance_checklist.html#contrib-pull-request-acceptance-checklist) seems to be an entirely different doc.. this doesn't have to be part of this PR, but I'd like to add it to the JIRA that tracks the `How to contribute a new Expectation to Great Expectations` how-to-guide